recipe/meta.yaml

```diff
@@ -1,15 +1,17 @@
+{% set name = "PyWavelets" %}
 {% set version = "1.0.3" %}
 
 package:
-  name: pywavelets
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/PyWavelets/pywt/archive/v{{ version }}.tar.gz
-  sha256: b152d4b551e3b454217c907e8722f7d8dc0adf104e6b3a8657d07e4020a990f2
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
You don't need the `fn`.

Reply:
Thanks. I have made the suggested change